packages/seacas/libraries/ioss/cmake/Dependencies.cmake

```diff
@@ -6,7 +6,7 @@ TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
 else()
 TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   LIB_OPTIONAL_PACKAGES SEACASExodus Pamgen Zoltan Kokkos
-  LIB_OPTIONAL_TPLS HDF5 CGNS ParMETIS Faodel Cereal DLlib Pthread DataWarp ADIOS2 Catalyst2 GTest
```

Comment:
FYI: After the merge of the related PR #11156, the next snapshot of SEACS will pull in:

https://github.com/sandialabs/seacas/blob/master/packages/seacas/libraries/ioss/cmake/Dependencies.cmake

which has:

```
TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
  LIB_OPTIONAL_PACKAGES SEACASExodus Pamgen Zoltan Kokkos
  LIB_OPTIONAL_TPLS HDF5 CGNS ParMETIS Faodel Cereal DLlib Pthread DataWarp ADIOS2 Catalyst2 ${SEACAS_GTest_TPL_name}
)
```

That will make it so that this file will not list the `GTest` TPL when configuring SEACAS under Trilinos.

But for now, the changes in this particular PR are a good short-term solution (because removing these listed TPLs is what was happening anyway).